modules/custom-status/rest/custom-status-endpoint.php

```diff
@@ -43,16 +44,22 @@ public static function register_routes() {
 				],
 
 				// Optional parameters
-				'description'        => [
+				'description'          => [
 					'default'           => '',
 					'sanitize_callback' => function ( $param ) {
 						return stripslashes( wp_filter_nohtml_kses( trim( $param ) ) );
 					},
 				],
-				'is_review_required' => [
-					'default'           => false,
+				'required_user_logins' => [
```

Comment:
If we go with the idea of calling the user endpoint via wp api, I think we should switch this to be user IDs instead. Then, we can simply validate they exist with a quick function call or query and move on to using them. We would save another call to convert them to IDs as well. It'll be a lot quicker to do a quick lookup on the IDs.

I can see right now a delay in the whole process, compared to before and that's only with 2 users on a status out of a total user set of 4 users.